packages/faucet/src/api/requestparser.spec.ts

```diff
@@ -3,7 +3,15 @@ import { RequestParser } from "./requestparser";
 describe("RequestParser", () => {
   it("can process valid credit request with denom", () => {
     const body = { address: "abc", denom: "utkn" };
-    expect(RequestParser.parseCreditBody(body)).toEqual({ address: "abc", denom: "utkn", amount: 1 });
+    expect(RequestParser.parseCreditBody(body)).toEqual({ 
+      address: "abc", 
+      denom: "utkn", 
+      amount: 1,
+      email: "test@example.com",
+      marketingOptin: false,
+      name: "John Doe",
+      company: "F Corp"
```

Comment:
Is country needed here as well?

Reply:
Not really - we are parsing request body here and we get the country from headers